docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

```diff
@@ -129,15 +128,15 @@ environment by using the use-local flag:
 
 .. code-block:: bash
 
-    conda install --use-local pyinstrument
+    conda install --use-local click
 
-Now verify that Pyinstrument installed successfully:
+Now verify that click installed successfully:
 
 .. code-block:: bash
 
     conda list
```

Comment:
Would be nice to have a screenshot/example of what the user is expected to see here